functions/Test-DbaDatabseBestPractice

```diff
@@ -0,0 +1,305 @@
+Function Test-DbaDatabaseBestPractice
+{
+<#
+.SYNOPSIS
+Tests all databases on a SQL Server instance to see if they are following best practices.
+	
+.DESCRIPTION
+Returns Database name and $true or $false by default for one server. Returns Server name, Database name and IsBestPractice for more than one server.
+	
+Specify -Detailed for details.
+	
+References:
+https://technet.microsoft.com/en-us/library/dd758814(v=sql.100).aspx - "The performance question here is usually not one of correlation per the formula, but whether the cluster size ..has been explicitly defined at 64 KB, which is a best practice for SQL Server."
+http://tk.azurewebsites.net/2012/08/
+	
+.PARAMETER ComputerName
+The SQL Server (or server in general) that you're connecting to. The -SqlServer parameter also works.
+	
+.PARAMETER SqlCredential
+If you want to use SQL Server Authentication to connect.
+
+.PARAMETER Detailed
+Show a detailed list.
+
+.PARAMETER Databases
+Test specific databases. This parameter is auto-populated by databases on the SQL Server.
+
+.NOTES
+Original Author: Mötz Jensen (@splaxi)
+	
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/Test-DbaDatabaseBestPractice
+
+.EXAMPLE
+Test-DbaDatabaseBestPractice -SqlServer sqlserver2014a
+
+To return true or false for any database not being compliant with the best practice
+
+.EXAMPLE
+Test-DbaDatabaseBestPractice -SqlServer sqlserver2014a -Databases db1, db2
+
+To return true or false for databases db1 and db2 not being compliant with the best practice
+
+.EXAMPLE   
+Test-DbaDatabaseBestPractice -SqlServer sqlserver2014 -Detailed
+	
+To return detailed information about databases and their best practice compliance 
+	
+#>
+	[CmdletBinding(SupportsShouldProcess = $true)]
```

Comment:
Since you are not making changes in this cmdlet, you can remove the SupportsShouldProcess. This is only needed I believe to support "-WhatIf" and "-Confirm", which is not needed for commands that are just reading and outputting information.